sdk/lib/atl/atlwin.h

```diff
@@ -20,6 +20,8 @@
 
 #pragma once
 
+#include <atlstr.h>
```

Comment:
When compiling we get an error concerning exceptions that need to be enabled....

Reply:
(Note that original MS atl doesn't include this header in that file).